style.css

```diff
@@ -6,180 +6,163 @@
-  content: "";
-  position: absolute;
-  display: block;
-  width: 100%;
-  height: 100%;
-  background: black;
-}
-
-label .hamburger:before { top: -10px; }
-
-label .hamburger:after { bottom: -10px; }
-
-label input { display: none; }
-
-/*style for close button and whitebackground of menu*/
-label input:checked + .menu {
-  box-shadow: 0 0 0 100vw #FFF,  0 0 0 100vh #FFF;
-  border-radius: 0;
-  
-}
-
-label input:checked + .menu .hamburger {
-  -webkit-transform: rotate(45deg);
-  transform: rotate(45deg);
-  
-}
-
-label input:checked + .menu .hamburger:after {
-  -webkit-transform: rotate(90deg);
-  transform: rotate(90deg);
-  bottom: 0;
-  
-}
-
-label input:checked + .menu .hamburger:before {
-  -webkit-transform: rotate(90deg);
-  transform: rotate(90deg);
-  top: 0;
-  
-}
-
-label input:checked + .menu + ul { opacity: 1; }
-
-/* link style */
-label ul {
-  font-family: 'Lexend', sans-serif; 
-  color: black;
-  font-size: 18pt;
-  font-weight: bold;
-  letter-spacing: 0.04em;
-  z-index: 200;
-  position: absolute;
-  top: 50%;
-  left: 50%;
-  -webkit-transform: translate(-50%, -50%);
-  transform: translate(-50%, -50%);
-  opacity: 0;
-  -webkit-transition: .25s 0s ease-in-out;
-  transition: .25s 0s ease-in-out;
-
-}
-
-label a {
-  margin-bottom: 1em;
-  display: block;
-  color: black;
-  text-decoration: none;
-  
-}
-
-/*style for burger menu ends*/
+	background-color: blue;
+	/* Create the parallax scrolling effect */
+ 	background-attachment: fixed;
+  	background-position: center;
+ 	background-repeat: no-repeat;
+  	background-size: cover;
+	overflow-x: hidden;
+	margin: 0;
+	padding: 0;
+}
+
+
+/*style for nav menu starts*/
+.navbar{
+	box-sizing: border-box;
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+	background-color: white;
+}
+
+.navbar-links ul{
+	margin: 0;
+	padding: 0;
+	display: flex;
+}
+
+.navbar-links li{
+	list-style: none;
+}
+
+.navbar-links li a{
+	text-decoration: none;
+	padding: 1rem;
+	display: block;
+}
+
+/* burger menu icon that displays on smaller screen sizes*/
+.toggle-button{
+	position: absolute;
+	top: .75rem;
+	right: 1rem;
+	display: none; /* not default so it only displays on smaller screen sizes*/
+	flex-direction: column;
+	justify-content: space-between;
+	width: 30px;
+	height: 21px;
+}
+
+.toggle-button .bar{
+	width: 100%;
+	height: 3px;
+	border-radius: 10px;
+	background-color: black;
+}
+
+/* media query that defines what width the navbar changes into a burger menu*/
+@media (max-width: 600px){
+	.toggle-button{
+		display: flex;
+	}
+	
+	.navbar-links{
+		display: none;
+		width: 100%;
+	}
+	
+	.navbar{
+		flex-direction: column;
+		align-items: flex-start;
+	}
+	
+	.navbar-links ul{
+		width: 100%;
+		flex-direction: column;
+	}
+	
+	.navbar-links li{
+		text-align: center;
+	}
+	
+	.navbar-links.active{
+		display: flex;
+	}
+}
+/*style for nav menu ends*/
 
 /* style for image banner - used in query pages */
 .banner{
+	margin-top: -20px;
```

Comment:
made banner smaller and have no space between elements so that you could see all elements at once